src/devices/Pcf8563/Pcf8563.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Pcf8563
+{
+    /// <summary>
+    /// Realtime Clock PCF8563
+    /// </summary>
+    public class Pcf8563 : IDisposable
+    {
+        private I2cDevice _i2cDevice = null;
+
+        /// <summary>
+        /// PCF8563 Default I2C Address
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x51;
+
+        /// <summary>
+        /// DS3231 DateTime
```

Comment:
Looks like copy paste from other binding. Also, could we just rename it to somehting like `The device's <see cref="DateTime"/>`

Reply:
My fault😅. Over time, I really can't remember how the code was named, such as `ReadTime` or `GetTime`...